packages/bindgen/spec.yml

```diff
@@ -287,12 +287,12 @@ enums:
       - Error
       - Superseded
       - AwaitingMark
-  WatchStreamState:
-    cppName: app::WatchStream::State
-    values:
-      - NEED_DATA
-      - HAVE_EVENT
-      - HAVE_ERROR
+  # WatchStreamState:
+  #   cppName: app::WatchStream::State
+  #   values:
+  #     - NEED_DATA
+  #     - HAVE_EVENT
+  #     - HAVE_ERROR
```

Comment:
Why? This is needed for watch support.

Actually, we should look into reverting all of the changes in this file.